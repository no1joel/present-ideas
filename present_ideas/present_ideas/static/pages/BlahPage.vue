<template>
  <h1>Blah!</h1>
</template>

<script>
export default {};
</script>