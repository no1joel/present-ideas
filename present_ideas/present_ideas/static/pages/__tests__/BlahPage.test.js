import { render } from "@testing-library/vue";
import BlahPage from "../BlahPage.vue";

describe("BlahPage", () => {
  it("is a good place to write dummy tests", () => {
    expect(true).toBe(true);
  });

  it("matches snapshot", () => {
    expect(render(BlahPage).container).toMatchSnapshot();
  });
});
