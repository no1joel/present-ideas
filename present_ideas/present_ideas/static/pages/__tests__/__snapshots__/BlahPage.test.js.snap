// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlahPage matches snapshot 1`] = `
<div>
  <h1>
    Blah!
  </h1>
</div>
`;
